testdata/baselines/reference/submoduleAccepted/conformance/importTag15(module=esnext).errors.txt.diff

```diff
@@ -0,0 +1,22 @@
+--- old.importTag15(module=esnext).errors.txt
++++ new.importTag15(module=esnext).errors.txt
+@@= skipped -0, +0 lines =@@
+ 1.js(1,30): error TS2857: Import attributes cannot be used with type-only imports or exports.
++1.js(2,27): error TS2307: Cannot find module './0' or its corresponding type declarations.
+ 1.js(2,33): error TS2857: Import attributes cannot be used with type-only imports or exports.
+
+
+ ==== 0.ts (0 errors) ====
+     export interface I { }
+     
+-==== 1.js (2 errors) ====
++==== 1.js (3 errors) ====
+     /** @import { I } from './0' with { type: "json" } */
+                                  ~~~~~~~~~~~~~~~~~~~~~
+ !!! error TS2857: Import attributes cannot be used with type-only imports or exports.
+     /** @import * as foo from './0' with { type: "json" } */
++                              ~~~~~
++!!! error TS2307: Cannot find module './0' or its corresponding type declarations.
```

Comment:
This is odd; wonder why this appears.

Reply:
There are a number of bad effects from #791 and #970 that I’m working through.